### PR TITLE
4.4 workload identity

### PR DESCRIPTION
**Pull request by renetapopova**
Cherry-picks #1131 and updates the version and the yaml file.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1157.surge.sh

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc [hunk 1]
```diff
@@ -165,36 +169,117 @@ backup:
...
+For more information on setting up a service account with workload identity, see https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Google Kubernetes Engine (GKE) -> Use Workload Identity], https://docs.aws.amazon.com/eks/latest/userguide/associate-service-account-role.html[Amazon EKS -> Configuring a Kubernetes service account to assume an IAM role], and https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview?tabs=go[Microsoft Azure -> Use Microsoft Entra Workload ID with Azure Kubernetes Service (AKS)].
+
+To configure the Neo4j backup pod to use a Kubernetes service account with workload identity, set `serviceAccountName` to the name of the service account to use.
+For Azure deployments, you also need to set the `azureStorageAccountName` parameter to the name of the Azure storage account, where the backup files will be uploaded.
+For example:
+
+[.tabbed-example]
+=====
+[.include-with-gke]
+======
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:187]: in 4.4 we dont have consistencyCheck attribute..this is the correct yaml

```
neo4j:
  image: "neo4j/helm-charts-backup"
  imageTag: "4.4.27"
  jobSchedule: "* * * * *"
  successfulJobsHistoryLimit: 3
  failedJobsHistoryLimit: 1
  backoffLimit: 3
backup:
  bucketName: "my-bucket"
  databaseAdminServiceName:  "standalone-admin" #This is the Neo4j Admin Service name.
  database: "neo4j,system"
  cloudProvider: "gcp"
  secretName: ""
  secretKeyName: ""

serviceAccountName: "demo-service-account"
```

**Review by harshitsinghvi22 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc [hunk 2]
```diff
@@ -165,36 +169,111 @@ backup:
...
+  backoffLimit: 3
+
+backup:
+  bucketName: "my-bucket"
+  databaseAdminServiceName:  "standalone-admin"
+  database: "neo4j,system"
+  cloudProvider: "aws"
+  secretName: ""
+  secretKeyName: ""
+ checkConsistency: true
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:230]: extra space needed before checkConsistency

**Review by harshitsinghvi22 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc (hunk 2 above)
Comment (renetapopova) [reply modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:230]: ah, yes

**Pull request merged**